src/docs/user.yaml

```diff
@@ -28,3 +28,27 @@ paths:
           description: 'Unauthorized (one needs to be logged in)'
         '500':
           description: 'Internal server error (ID is not a valid ObjectId or a server error occurred)'
+    delete:
+      tags:
+        - User
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          required: true
+          description: 'ID of the user'
+          examples:
+            'real ID of an available user':
+              value: '62eb038c20b4f5b0bd8c7676'
+            'some fake ObjectId':
+              value: 'fake5c280262c62f56124dfd'
```

Comment:
maybe you can add another value something like '62eb038c20b4f5b0bd8c7xxx' so that we can get a 404 error